config.py

```diff
@@ -1,29 +1,31 @@
 import os
-import sys
-from typing import Optional
 
-from ape import networks, accounts
+from ape import accounts
 
 
-def get_network_name() -> Optional[str]:
-    # It is None at moment of config.py network initialization
-    if networks.active_provider is None:
-        network_index = sys.argv.index("--network")
-        if network_index + 1 < len(sys.argv):
-            network_arg = sys.argv[network_index + 1]
-            _, name, provider = network_arg.split(":")
-            return name
-        return None
+NETWORK_ENV_VAR = "NETWORK"
+ALLOWED_NETWORKS = ["mainnet", "goerli", "holesky"]
```

Comment:
It's often better to store network names in a var to avoid typos.

```python
MAINNET = "mainnet"
GOERLI = "goerli"
HOLESKY = "holesky"
ALLOWED_NETWORKS = [MAINNET, GOERLI, HOLESKY]
```